cloud/amazon/ec2_vpc_igw.py

```diff
@@ -133,14 +137,14 @@ def main():
 
     if region:
         try:
-            connection = connect_to_aws(boto.ec2, region, **aws_connect_params)
+            connection = connect_to_aws(boto.vpc, region, **aws_connect_params)
         except (boto.exception.NoAuthHandlerFound, StandardError), e:
             module.fail_json(msg=str(e))
     else:
         module.fail_json(msg="region must be specified")
 
     vpc_id = module.params.get('vpc_id')
-    state = module.params.get('state', 'present')
+    state = module.params.get('state')
```

Comment:
If this change is done because of the `default='present'` declaration above, it would be more clear to do `module.params['state']` to indicate that you are not expecting `None` back here.